list_sort.py

```diff
@@ -0,0 +1,29 @@
+from collections import defaultdict
+
+def list_sort():
+               
+    listtobesorted = ([2, 0, 6, 5, 1, 7, 'z', 'a'])
```

Comment:
So, how about instead of hardcoding the list to be sorted, you pass it as a method parameter. Something like `def list_sort(array_to_be_sorted)`. By doing that you make it such that the `list_sort` method can be called with any array that needs to be sorted. Does this make sense?